### PR TITLE
Issues/42 merge presto pizza/restyle app

### PR DESCRIPTION
**Pull request by darjanin**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -214,9 +214,10 @@ export function parseOrders() {
           const pizzaSize = order.match(/\d+/g)[1];
 
           if (!pizzaSize || pizzaSize === '33') {
-            pizza[pizzaNum] = get(pizza, pizzaNum, 0) + 1;
+            presto.pizza[pizzaNum] = get(presto.pizza, pizzaNum, 0) + 1;
           } else {
-            pizza[`${pizzaNum} veľkosti ${pizzaSize}`] = get(pizza, `${pizzaNum} veľkosti ${pizzaSize}`, 0) + 1;
+            presto.pizza[`${pizzaNum} veľkosti ${pizzaSize}`] =
+              get(presto.pizza, `${pizzaNum} veľkosti ${pizzaSize}`, 0) + 1;
```
Comment (misogl) [inline src/utils.js]:
```js
const key = (!pizzaSize || pizzaSize === '33')
  ? pizzaNum
  : `${pizzaNum} veľkosti ${pizzaSize}`
```
and then use the `key`.

File: views/includes/head.pug [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
     rel='stylesheet',
-    href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css',
-    integrity='sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u',
+    href='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css',
+    integrity='sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb',
+    crossorigin='anonymous'
+  )
+  link(rel='stylesheet', href='public/css/style.css')
+  script(
+    src='https://code.jquery.com/jquery-3.2.1.slim.min.js'
```
Comment (misogl) [inline views/includes/head.pug:15]: Do we use `jQuery` and `popper`?

File: views/includes/foodWontArrive.pug [hunk 3]
```diff
@@ -1,21 +1,16 @@
...
-  div
-    a(href="/nopresto")
-      button(
-        class='btn btn-warning btn-responsive',
+h3='Nepríde jedlo :-('
+.row
+  .col
+    .btn-group
+      a.btn.btn-warning(
+        href="/prenoprestosto"
```
Comment (misogl) [inline views/includes/foodWontArrive.pug]: typo

File: public/css/style.css [hunk 4]
```diff
@@ -1,117 +1,35 @@
 body {
   background-image: url('../background.png');
```
Comment (misogl) [inline public/css/style.css:2]: Do you like the background? I think it could be improved.

**Review by misogl — CHANGES_REQUESTED** [review]

File: views/includes/head.pug (hunk 2 above)
Comment (darjanin) [reply views/includes/head.pug:15]: It's used on mobile version for the navbar toggler.

File: public/css/style.css (hunk 4 above)
Comment (darjanin) [reply public/css/style.css:2]: I don't mind it. It's not great but it doesn't distract me too much. And other options option will be plain color background.

File: views/includes/head.pug (hunk 2 above)
Comment (darjanin) [reply views/includes/head.pug:15]: For the Bootstrap. I don't want to implement it if it's part of bootstrap.

**Review by misogl — APPROVED** [review]

**Pull request merged**